ocs_ci/deployment/vmware.py

```diff
@@ -340,9 +340,13 @@ def deploy_prereq(self):
             self.kubeconfig = os.path.join(self.cluster_path, config.RUN.get('kubeconfig_location'))
 
             # git clone repo from openshift installer
+            # installer ( https://github.com/openshift/installer ) master and
+            # other branches (greater than release-4.4) structure has been
+            # changed. use master branch when ocs-ci is ready to incorporated
```

Comment:
I would rather use alwas the branch for versiom and have logic changed from specific version only.

Reply:
Updated the comment section